packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -59,7 +59,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     copyPaste?: boolean;
     readOnly?: boolean;
     filterProp?: boolean;
-    toolbar?: boolean;
+    toolbar?: false | { hideTopBar?: boolean; automaticTitleItem?: boolean };
```

Comment:
maybe rename `automaticTitleItem` to just `title`? (as the usage of `ToolbarAutomaticTitleItem` is an not-that-important implementation detail)